images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -15,101 +15,103 @@ function Get-GoVersion {
 }
 
 function Get-Cbindgen {
     $cbindgenVersion = Run-Command "cbindgen --version" | Take-Part -Part 1
-    return "Cbindgen $cbindgenVersion"
+    return $cbindgenVersion
 }
 
 function Get-Cargooutdated {
     $cargoOutdatedVersion = Run-Command "cargo outdated --version" | Take-Part -Part 1
-    return "Cargo-outdated $cargoOutdatedVersion"
+    return $cargoOutdatedVersion
 }
 
 function Get-Cargoaudit {
     $cargoAuditVersion = Run-Command "cargo-audit --version" | Take-Part -Part 1
-    return "Cargo-audit $cargoAuditVersion"
+    return $cargoAuditVersion
 }
 
 function Get-RustupVersion {
     $rustupVersion = Run-Command "rustup --version" | Select-Object -First 1 | Take-Part -Part 1
-    return "Rustup ${rustupVersion}"
+    return $rustupVersion
 }
 
 function Get-VcpkgVersion {
     $vcpkgVersion = Run-Command "vcpkg version" | Select-Object -First 1 | Take-Part -Part 5 | Take-Part -Part 0 -Delimiter "-"
     $commitId = git -C "/usr/local/share/vcpkg" rev-parse --short HEAD
-    return "Vcpkg $vcpkgVersion (build from master \<$commitId>)"
+    return "$vcpkgVersion (build from master \<$commitId>)"
 }
 
-function Get-GccVersion {
+function Get-GccVersions {
     $versionList = Get-ToolsetValue -KeyPath gcc.versions
     $versionList | Foreach-Object {
-        $version = Run-Command "gcc-${_} --version" | Select-Object -First 1
-        "$version - available by ``gcc-${_}`` alias"
+        $nameVersion = Run-Command "gcc-${_} --version" | Select-Object -First 1
+        $version = ($nameVersion -replace "^gcc-${_}").Trim() -replace '\).*$', ')'
+        return [ToolNode]::new("GCC ${_}", "$version - available by ``gcc-${_}`` alias")
     }
 }
 
-function Get-FortranVersion {
+function Get-FortranVersions {
     $versionList = Get-ToolsetValue -KeyPath gcc.versions
     $versionList | Foreach-Object {
-        $version = Run-Command "gfortran-${_} --version" | Select-Object -First 1
-        "$version - available by ``gfortran-${_}`` alias"
+        $nameVersion = Run-Command "gfortran-${_} --version" | Select-Object -First 1
+        $version = ($nameVersion -replace "^GNU Fortran").Trim() -replace '\).*$', ')'
+        return [ToolNode]::new("GNU Fortran ${_}", "$version - available by ``gfortran-${_}`` alias")
     }
 }
 
-function Get-ClangLLVMVersion {
+function Get-ClangLLVMVersions {
     $toolsetVersion = '$(brew --prefix llvm@{0})/bin/clang' -f (Get-ToolsetValue 'llvm.version')
     $locationsList = @("$((Get-Command clang).Source)", $toolsetVersion)
     $locationsList | Foreach-Object {
         (Run-Command "${_} --version" | Out-String) -match "(?<version>\d+\.\d+\.\d+)" | Out-Null
         $version = $Matches.version
-        "Clang/LLVM $version " + $(if(${_} -Match "brew") {"is available on ```'${_}`'``"} else {"is default"})
+        return [ToolNode]::new("Clang/LLVM$(if(${_} -Match "brew") {" (Homebrew)"})", "$version " + $(if(${_} -Match "brew") {"- available on ```'${_}`'``"} else {"- default"}))
```

Comment:
May be it will be clear to rewrite this function without loop?
```pwsh
function Get-ClangLLVMVersions {
    $clangVersionRegex = [Regex]::new("(?<version>\d+\.\d+\.\d+)")
    $defaultClangOutput = Run-Command "clang --version" | Out-String
    $defaultClangVersion = $clangVersionRegex.Match($defaultClangOutput).Groups['version'].Value

    $homebrewClangPath = '$(brew --prefix llvm@{0})/bin/clang' -f (Get-ToolsetValue 'llvm.version')
    $homebrewClangOutput = Run-Command "$homebrewClangPath --version" | Out-String
    $homebrewClangVersion = $clangVersionRegex.Match($homebrewClangOutput).Groups['version'].Value

    return @(
        [ToolNode]::new("Clang/LLVM", $defaultClangVersion),
        [ToolNode]::new("Clang/LLVM (Homebrew)", "$homebrewClangVersion - available on $homebrewClangPath")
    )
}
```

IMHO, it will look a bit clearer.

Also, I think we shouldn't add `default` postfix to the first item